src/cli/services/__tests__/linter.test.ts

```diff
@@ -554,81 +554,37 @@ describe('Linter service', () => {
 
   describe('$ref linting', () => {
     it('outputs errors occurring in referenced files', () => {
-      return expect(run(`lint ${draftRefSpec}`)).resolves.toEqual([
-        expect.objectContaining({
-          code: 'oas2-api-schemes',
-          message: 'OpenAPI host `schemes` must be present and non-empty array.',
-          path: [],
-          range: expect.any(Object),
-          source: expect.stringContaining('__tests__/__fixtures__/draft-ref.oas2.json'),
-        }),
-        expect.objectContaining({
-          code: 'oas2-schema',
-          message: '/info Property foo is not expected to be here',
-          path: ['info'],
-          range: {
-            end: {
-              character: 5,
-              line: 9,
-            },
-            start: {
-              character: 12,
-              line: 3,
-            },
-          },
-          source: expect.stringContaining('__tests__/__fixtures__/refs/info.json'),
-        }),
-        expect.objectContaining({
-          code: 'info-description',
-          message: 'OpenAPI object info `description` must be present and non-empty string.',
-          path: ['info', 'description'], // todo: relative path or absolute path? there is no such path in linted file, but there is such in spec when working on resolved file
-          range: {
-            end: {
-              character: 22,
-              line: 5,
-            },
-            start: {
-              character: 21,
-              line: 5,
+      return expect(run(`lint ${draftRefSpec}`)).resolves.toEqual(
+        expect.arrayContaining([
```

Comment:
@philsturgeon This change (switching from pure array equality to array "containing" equality is the change that bothers me the most.

If I've got some more time later today, I'll try and review why that was needed and if I can't find a better fix.

Reply:
No longer relevant